plugin/utiltools.h

```diff
@@ -21,6 +21,9 @@ class UtilTools: public QObject
     Q_INVOKABLE void restartNetwork(QJSValue successCallback = QJSValue::UndefinedValue,
                                     QJSValue errorCallback = QJSValue::UndefinedValue);
 
+    Q_INVOKABLE void restartBluetooth(QJSValue successCallback = QJSValue::UndefinedValue,
+                                    QJSValue errorCallback = QJSValue::UndefinedValue);
```

Comment:
```suggestion
    Q_INVOKABLE void restartBluetooth(QJSValue successCallback = QJSValue::UndefinedValue,
                                      QJSValue errorCallback = QJSValue::UndefinedValue);
```
To follow the convention in the rest of the code, it looks like items in subsequent lines should align with the opening brackets.